syntax = "proto3";

service AlarmSensor {
  rpc SendData (Data) returns (Empty) {}
}

message Empty {}

message Data {
  int32 id = 1;
  int32 pir_data = 2;
  int32 ultrasonic_data = 3;
  // Hard location of PIR sensor
  // Hard location of US sensor
}